frontend/components/commons/forms/questions-form/QuestionsForm.component.vue

```diff
@@ -443,12 +443,39 @@ export default {
 
               switch (componentType) {
                 case COMPONENT_TYPE.SINGLE_LABEL:
+                  formattedOptions = formattedOptions.map((option) => {
+                    if (option.value_in_api === newResponse.value) {
```

Comment:
Two objects are almost the same, is it possible to make the condition at object value level?